tests/foreman/api/test_docker.py

```diff
@@ -603,3 +603,159 @@ def test_positive_remove_docker_repo_ccv(
         assert ak.content_view.id == comp_content_view.id
         ak.content_view = None
         assert ak.update(['content_view']).content_view is None
+
+
+class TestPodman:
+    """Tests specific to using podman push/pull on Satellite
+
+    :CaseComponent: Repositories
+
+    :team: Phoenix-content
+    """
+
+    @pytest.fixture(scope='class')
+    def enable_podman(module_product, module_target_sat):
+        """Enable base_os and appstream repos on the sat through cdn registration and install podman."""
+        module_target_sat.register_to_cdn()
+        if module_target_sat.os_version.major > 7:
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhel_bos']['id'])
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhel_aps']['id'])
+        else:
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhscl']['id'])
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhel']['id'])
+        result = module_target_sat.execute(
+            'dnf install -y --disableplugin=foreman-protector podman'
+        )
+        assert result.status == 0
+
+    def test_podman_push(self, module_target_sat, module_product, module_org, enable_podman):
+        """Push a small and large container image to Pulp, and verify the results
+
+        :id: 488adc49-899e-4739-8bca-0cd255da63ae
+
+        :steps:
+            1. Using podman, pull a small and large image from the fedoraproject registry
+            2. Push one image to pulp using the /org_label/product_label format
+            3. Push the other image to pulp using the /id/org_id/product_id format
+
+        :expectedresults: A docker repository is created for both images. Both images are published in pulp,
+        as well as in separate repositories in the given product. All fields on both repositories contain correct information.
```

Comment:
i think you need to add some indentation here
```suggestion
            as well as in separate repositories in the given product. All fields on both repositories contain correct information.
```